docs/fileio_usage.rst

```diff
@@ -3,60 +3,104 @@ Usage
+    start_date = datetime.date.today()  # This needs to be a datetime object
+    probe = 'neutron'
+    facility = 'ESS'
+    proposalID = 999999
+    doi = f'10.5286/ISIS.E.RB{99999}'
+
+    experiment = Experiment(title, instrument, start_date, probe, facility, proposalID, doi)
 
 Full details of the different components that can be populated can be found in the `documentation`_ here or in the `file format specification`_.
 Note that this specification includes information regarding the **required** and optional components to be included for a file to be considered a **valid** .ort file.
-It is not possible to write a .ort file without defining the columns present in the dataset, in this example we will have four columns of data, namely q, R, dR and dq (the final column is a description of the resolution function). 
-Columns are defined as follows, using the :code:`orsopy.fileio.base.Column` and :code:`orsopy.fileio.base.ErrorColumn` class objects (note that there are other `base classes`_ that can be used for a variety of objects).
+What we show here is not the minimum needed to write an `.ort file, rather the minium to fill out all the major fields with information which is likely to be available at time of writing.
+
+We now write out the user and sample details, but note that the sample can be alot more descriptive than just a name.
+
+.. code-block:: python
+
+    user_name = 'Jos Cooper'
+    user_affil = 'ESS'
+
+    user = Person(user_name, user_affil)
+
+    # Now fill the sample details
+    sample_name = 'Silicon'
+
+    sample = Sample(sample_name)
+
+    # The measurement details needs some instrument settings assigned first
+    angle = 0.3  # This can be a float, or a range, though currently not a list
+    wavelength = (1.8, 8.8)  # Again a float or range
+
+    instrument_settings = InstrumentSettings(angle, wavelength)
+
+    # Now add the instrument settings to the names of the files used
+    files = ['test_data.raw']  # A list of all raw files used reduced
+
+    measurment = Measurement(instrument_settings, files)
+
+We can now wrap together the user, experiment, sample and measurement information into a single object.
+
+.. code-block:: python
+    data_source_info = DataSource(user, experiment, sample, measurment)
+
+It is not possible to write a .ort file without defining the columns present in the dataset, in this example we will have four columns of data, namely q, R, dR and dq (the final column is a description of the resolution function).
+Columns are defined as follows, using the :code:`orsopy.fileio.base.Column` and :code:`orsopy.fileio.base.ErrorColumn` class objects.
+
+.. code-block:: python
 
-.. code-block:: python 
-    
     q_column = fileio.base.Column(name='Qz', unit='1/angstrom', physical_quantity='wavevector transfer')
+    # Note that the unit is not fixed to 1/angstrom by the standard, but currently the data loader interprets all values as 1/angstrom irrespective of what is stated here
```

Comment:
Is this true?

Reply:
sadly yes. there is no unit processing that I could find in the data loader, and all analysis packages currently read data specified with 1/nanometer as 1/angstrom

Reply:
oh, but no!
I was just not using the right unit!!!

Reply:
`1/nm` not `1/nanometer`